webviewer/src/pages/Signup/components/Form/SignUpForm.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useUserActions } from 'state';
+import { FormField, FormFieldset } from 'components/_shared/Form';
+import { Button } from 'components/_shared/Button';
+import { ROUTES } from '_constants';
+import { validationSchema, _initialValues } from './formValues';
+import { StyledLink, StyledMessage, StyledForm } from './SignUp.styles';
+
+const formMessage = (
+  <>
+    <StyledMessage>Already have an account? </StyledMessage>
+    <StyledLink to={ROUTES.AUTH}>Login</StyledLink>
+  </>
+);
+const formButton = (
+  <Button key='submit' type='submit' variant='primary' fullWidth>
+    Sign up
+  </Button>
+);
+
+export const SignUpForm = ({ initialValues = {}, setEmail, ...props }) => {
+  const { isLoading, error, registerUser } = useUserActions();
+
+  const onSubmit = async values => {
+    await registerUser(values);
+    setEmail(values.email);
+  };
```

Comment:
if we try to imitate blocking the API on registration user, UI shows infinite loader overlay and form becomes disabled. 
Need to handle error in this case and if you have no error in response, you need to show some default text

![image](https://user-images.githubusercontent.com/23707448/221559013-d394084c-1a2e-410a-8b33-1ed4f7e99785.png)